mavros_msgs/msg/OrbitStatus.msg

```diff
@@ -0,0 +1,10 @@
+uint64 timestamp # time since system start (microseconds)
+float32 radius   # Radius of the orbit circle. Positive values orbit clockwise, negative values orbit counter-clockwise. [m]
+uint8 frame      # The coordinate system of the fields: x, y, z.
+float64 x        # X coordinate of center point. Coordinate system depends on frame field: local = x position in meters * 1e4, global = latitude in degrees * 1e7.
+float64 y        # Y coordinate of center point. Coordinate system depends on frame field: local = y position in meters * 1e4, global = latitude in degrees * 1e7.
+float32 z        # Altitude of center point. Coordinate system depends on frame field.
+uint8 state # See enum ORBIT_EXECUTION_STATE.
+uint8 ORBIT_EXECUTION_STATE_APPROACH_TO_CIRCLE = 0
+uint8 ORBIT_EXECUTION_STATE_STARTED = 1
+uint8 ORBIT_EXECUTION_STATE_FINISHED = 2  # Meaningless for infinite orbits.
```

Comment:
Pleasedefine constants before message fields.

Reply:
Also usually you could drop enum prefix like `ORBIT_EXECUTION_`.

Reply:
Again, my intent is to keep API exactly aligned wrt https://github.com/mavlink/mavlink/pull/1622/files#diff-9ef151b30f09d6d4f130434d941db479c3411ec15eca23580c5587b2c3b59ecdR707-R721.

Reply:
@tahsinkose please use ROS standard types.